Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 plugin "bundler-inject", "~> 1.1"
 require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundler-inject") rescue nil
 
-gem 'insights-api-common',  '~> 3.9'
+gem 'insights-api-common',  '=  3.8'
```

Comment:
Hoping the `~> 3.9` wasn't for any features only available from 3.9 on.

Reply:
Looking at the failing specs we need at least v3.9 to deal with the GraphQL changes.

Reply:
Pinning to 3.9 looks like its working!